manager/manager.go

```diff
@@ -509,6 +533,22 @@ func (m *manager) destroyContainer(containerName string) error {
 		})
 	}
 	glog.Infof("Destroyed container: %q (aliases: %v, namespace: %q)", containerName, cont.info.Aliases, cont.info.Namespace)
+
+	contRef, err := cont.handler.ContainerReference()
+	if err != nil {
+		return err
+	}
+
+	newEvent := &events.Event{
+		ContainerName: contRef.Name,
+		EventData:     cont,
```

Comment:
The information from cont.handler.GetSpec() pertains more to the container when it was alive, but we could still return it here as event data in case the user wanted to see information about the container before it was stopped.